README.md

```diff
@@ -1 +1,163 @@
+You can install the PHP SDK with Composer, either run `composer require microsoft/microsoft-graph-core`, or edit your `composer.json` file:
+```
+{
+    "require": {
+        "microsoft/microsoft-graph": "^0.1.0"
+    }
+}
+```
+## Get started with Microsoft Graph
+
+### Register your application
+
+Register your application to use the Microsoft Graph API using [Microsoft Azure Active Directory](https://portal.azure.com/#blade/Microsoft_AAD_RegisteredApps/ApplicationsListBlade) in your tenant's Active Directory to support work or school users for your tenant, or multiple tenants.
+
+### Authenticate with the Microsoft Graph service
+
+The Microsoft Graph SDK for PHP does not include any default authentication implementations. The [`thephpleague/oauth2-client`](https://github.com/thephpleague/oauth2-client) library will handle the OAuth2 flow for you and provide a usable token for querying the Graph.
+
+To authenticate as an application you can use the [Guzzle HTTP client](http://docs.guzzlephp.org/en/stable/), which comes preinstalled with this library, for example like this:
+```php
+$guzzle = new \GuzzleHttp\Client();
+$url = 'https://login.microsoftonline.com/' . $tenantId . '/oauth2/token?api-version=1.0';
+$token = json_decode($guzzle->post($url, [
+    'form_params' => [
+        'client_id' => $clientId,
+        'client_secret' => $clientSecret,
+        'resource' => 'https://graph.microsoft.com/',
+        'grant_type' => 'client_credentials',
+    ],
+])->getBody()->getContents());
+$accessToken = $token->access_token;
+```
+For an integrated example on how to use Oauth2 in a Laravel application and use the Graph, see the [PHP Connect Sample](https://github.com/microsoftgraph/php-connect-sample).
+
+### Call Microsoft Graph using the v1.0 endpoint and models
+
+The following is an example that shows how to call Microsoft Graph.
+
+```php
+use Microsoft\Graph\Graph;
+use Microsoft\Graph\Model;
+
+class UsageExample
+{
+    public function run()
+    {
+        $accessToken = 'xxx';
+
+        $graph = new Graph();
+        $graph->setAccessToken($accessToken);
+
+        $user = $graph->createRequest("GET", "/me")
+                      ->setReturnType(Model\User::class)
+                      ->execute();
+
+        echo "Hello, I am {$user->getGivenName()}.";
+    }
+}
+```
+
+### Call Microsoft Graph using the beta endpoint and models
+
+The following is an example that shows how to call Microsoft Graph.
+
+```php
+use Microsoft\Graph\Graph;
+use Beta\Microsoft\Graph\Model as BetaModel;
+
+class UsageExample
+{
+    public function run()
+    {
+        $accessToken = 'xxx';
+
+        $graph = new Graph();
+        $graph->setAccessToken($accessToken);
+
+        $user = $graph->setApiVersion("beta")
+                      ->createRequest("GET", "/me")
+                      ->setReturnType(BetaModel\User::class)
+                      ->execute();
+
+        echo "Hello, I am $user->getGivenName() ";
+    }
+}
+```
```

Comment:
Should we remove this section for this package since we don't have  models?
Or remove `setReturnType()` from the snippets & demo how to use the raw `GraphResponse` object?

Reply:
We should demo how to use the GraphResponse. There will be customers who will use this approach. Certainly remove references to models, but provide links to the model repos.